src/order-history/sagas.js

```diff
@@ -0,0 +1,30 @@
+import { call, put, takeEvery } from 'redux-saga/effects';
+
+// Actions
+import {
+  FETCH_ORDERS,
+  fetchOrdersBegin,
+  fetchOrdersSuccess,
+  fetchOrdersFailure,
+  fetchOrdersReset,
+} from './actions';
+
+// Services
+import * as OrdersApiService from './services';
+
+export function* handleFetchOrders() {
+  try {
+    yield put(fetchOrdersBegin());
+    const result = yield call(OrdersApiService.getOrders);
+    yield put(fetchOrdersSuccess(result));
+    yield put(fetchOrdersReset());
+  } catch (e) {
+    // LoggingService.logAPIErrorResponse(e);
```

Comment:
Figure you're just deferring it, but we'll want error logging and all that, obvs.